src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -603,6 +606,8 @@ private void errorHandler(RoutingContext routingContext) {
             }
         } else if (routingContext.statusCode() == HttpResponseStatus.NOT_FOUND.code()) {
             message = HttpResponseStatus.NOT_FOUND.reasonPhrase();
+        } else if (routingContext.statusCode() == HttpResponseStatus.NOT_IMPLEMENTED.code()) {
+            message = HttpResponseStatus.NOT_IMPLEMENTED.reasonPhrase();
```

Comment:
What are we using the not-implemented code for? Not implemented is not the same as not enabled.